pkg/apis/serving/v1alpha1/route_validation.go

```diff
@@ -105,3 +108,15 @@ func (rs *RouteSpec) Validate(ctx context.Context) *apis.FieldError {
 	}
 	return errs
 }
+
+func validateAnnotations(annotations map[string]string) *apis.FieldError {
+	disableAutoTLS := annotations[networking.DisableAutoTLSLabelKey]
```

Comment:
just to be a total pedant, the constant is called `DisableAutoTLSLabelKey` but we're expecting this to be set as an annotation? I think annotation seems right here, but then the constant name is wrong :(.

Reply:
I didn't realize these were different things 😅  time to submit another PR to the networking repo...